authenticators/cf_authenticator.go

```diff
@@ -83,7 +84,16 @@ func (cfa *CFAuthenticator) Authenticate(metadata ssh.ConnMetadata, password []b
 		return nil, err
 	}
 
-	processGuid, err := cfa.checkAccess(logger, appGuid, index, string(cred))
+	parts := strings.Split(cred, " ")
+	tokenString := parts[1]
```

Comment:
in general this should not happen.  There is already checks to make sure the response from UAA is valid and correct.  But you are right this does assume it's a 3 part token which is not really validated.  I will add a test for it and return "AuthenicationFailedErr" if it is not correct.